web/src/scripts/webflow/licenseSync.mjs

```diff
@@ -205,8 +205,40 @@ const createNewLicenses = async () => {
   console.info(`Created a total of ${newLicenses.length} licenses`);
 };
 
+const getUnUsedLicenses = async () => {
+  const localLicensesRaw = loadAllLicenses();
+  const localLicensesFiltered = localLicensesRaw.filter((it) => it.webflowId === undefined);
+  const localLicensesIds = new Set(localLicensesFiltered.map((i) => i.webflowId));
+
+  const webflowLicense = await getAllLicensesFromWebflow();
+
+  return webflowLicense.filter((item) => {
+    const haslicense = localLicensesIds.has(item.fieldData.id);
+    return !haslicense;
+  });
+};
+
+const deleteLicenses = async () => {
+  const licenses = await getUnUsedLicenses();
+  const deleteLicense = async (license) => {
+    console.info(`Attempting to delete ${license.fieldData.slug}`);
```

Comment:
[sand] As a person doing release or reading through release logs, I'd like to know _why_ a license is being deleted. Otherwise a message like "Attempting to delete cosmetology" would be unnecessarily frightening.